### ISSUE TITLE
User Agent back to default once app restart

### ISSUE THREAD

**Issue by energetictree** [andrefmsilva/iplaytvaio-bugs#60]
User agent is not permanently saved and revert back to default once application is restarted.

**Comment by Shineson1001** [comment]
Yes, it would be nice, if the setting "User-Agent" would be saved permanently.
You have to re-enter the value on every App start.
Or is it a pro feature?

I use the version v1.1.0(20220927.0)
